sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -15,12 +15,17 @@
 # limitations under the License.
 #
 
-"""A cross-language word-counting workflow."""
+"""A word-counting workflow that uses the SQL transform.
+
+Java 8 and docker must be available to run this pipeline, and the
```

Comment:
How about:

Java 8 must be available to run this pipeline and --experiments=use_runner_v2 flag must be passed for running on Dataflow.

Additionally you need to install Docker to run this pipeline from HEAD with a custom Java container. See `here <https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/kafkataxi/>` for more details.